meta/task_functions.py

```diff
@@ -69,19 +69,28 @@ def notification_mattermost(ti):
             hours = int(average_duration // 3600)
             minutes = int((average_duration % 3600) // 60)
             start_time = sorted(successes.keys())[-1].split(' ')[1][:-3]
+            # setting time to UTC+2
+            start_hour = int(start_time.split(':')[0])
```

Comment:
Wouldn't it be easier to convert it to datetimes to manipulate instead?

Reply:
Idk if it's easier, I guess more elegant indeed

Reply:
less error-prone I would say